### ISSUE TITLE
in container mode, ctrl-c of telepresence takes forever

### ISSUE THREAD

**Issue by richarddli** [telepresenceio/telepresence#337]
```
lost-cause:qotm richard$ telepresence --swap-deployment qotm-dev-rdl-end-to-end --docker-run --rm -it -v $(pwd):/service qotm-dev:latest
Volumes are rooted at $TELEPRESENCE_ROOT. See https://telepresence.io/howto/volumes.html for details.

Password:
Guessing that Services IP range is 10.11.240.0/20. Services started after this point will be inaccessible if are outside this range; restart telepresence if you can't access a new Service.

2017-10-26 13:52:49 QotM 1.3 INFO: initializing on 81792dde1c4f:5000
2017-10-26 13:52:49 QotM 1.3 INFO:  * Running on http://0.0.0.0:5000/ (Press CTRL+C to quit)
2017-10-26 13:52:49 QotM 1.3 INFO:  * Restarting with stat
2017-10-26 13:52:49 QotM 1.3 INFO: initializing on 81792dde1c4f:5000
2017-10-26 13:52:49 QotM 1.3 WARNING:  * Debugger is active!
2017-10-26 13:52:49 QotM 1.3 INFO:  * Debugger PIN: 300-335-359
2017-10-26 13:53:03 QotM 1.3 DEBUG: GET /: session None, username None, handler statement
2017-10-26 13:53:03 QotM 1.3 INFO: 127.0.0.1 - - [26/Oct/2017 13:53:03] "GET / HTTP/1.1" 200 -
2017-10-26 13:53:36 QotM 1.3 DEBUG: GET /: session None, username None, handler statement
2017-10-26 13:53:36 QotM 1.3 INFO: 127.0.0.1 - - [26/Oct/2017 13:53:36] "GET / HTTP/1.1" 200 -
2017-10-26 13:54:26 QotM 1.3 DEBUG: GET /: session None, username None, handler statement
2017-10-26 13:54:26 QotM 1.3 INFO: 127.0.0.1 - - [26/Oct/2017 13:54:26] "GET / HTTP/1.1" 200 -
^Z^Z^C^C^C^C


^C^C^C


^C^C^C^C^F

^C

^C^C^C

Password:
Error in atexit._run_exitfuncs:
Traceback (most recent call last):
  File "/usr/local/Cellar/python3/3.6.3/Frameworks/Python.framework/Versions/3.6/lib/python3.6/subprocess.py", line 405, in run
    stdout, stderr = process.communicate(input, timeout=timeout)
  File "/usr/local/Cellar/python3/3.6.3/Frameworks/Python.framework/Versions/3.6/lib/python3.6/subprocess.py", line 830, in communicate
    stdout = self.stdout.read()
  File "/usr/local/bin/telepresence", line 2218, in shutdown
    raise SystemExit(0)
SystemExit: 0

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/Cellar/python3/3.6.3/Frameworks/Python.framework/Versions/3.6/lib/python3.6/subprocess.py", line 412, in run
    process.kill()
  File "/usr/local/Cellar/python3/3.6.3/Frameworks/Python.framework/Versions/3.6/lib/python3.6/subprocess.py", line 1590, in kill
    self.send_signal(signal.SIGKILL)
  File "/usr/local/Cellar/python3/3.6.3/Frameworks/Python.framework/Versions/3.6/lib/python3.6/subprocess.py", line 1580, in send_signal
    os.kill(self.pid, sig)
PermissionError: [Errno 1] Operation not permitted
Error in atexit._run_exitfuncs:
Traceback (most recent call last):
  File "/usr/local/bin/telepresence", line 944, in killall
    killer()
  File "/usr/local/bin/telepresence", line 1934, in kill
    runner.check_call(sudo + ["docker", "stop", "--time=1", name])
  File "/usr/local/bin/telepresence", line 384, in check_call
    raise CalledProcessError(retcode, args)
subprocess.CalledProcessError: Command '(['docker', 'stop', '--time=1', 'telepresence-1509025953-425097-9525'],)' returned non-zero exit status 1.
lost-cause:qotm richard$
```

**Comment by richarddli** [comment]
Is it possible for Ctrl-C to immediately signal a shutdown?

**Comment by ark3** [comment]
I suspect that once we fix #586, this will be resolved.

**Comment by donnyyung** [comment]
I believe this is no longer an issue in Telepresence 2. Here are the docs on how to install Telepresence (https://www.telepresence.io/docs/latest/install/), please re-open if you still see this issue in our latest version!
